scripts/docker/centos/stream8/Dockerfile

```diff
@@ -78,6 +79,13 @@ RUN dnf upgrade --refresh -y && dnf install -y \
 # Build dependency for libelektra-fuse
 RUN pip3 install wheel
 
+# see https://github.com/docker-library/golang/issues/225
+ENV XDG_CACHE_HOME=/tmp/.cache
+
+# see https://github.com/golang/go/wiki/InvalidFlag
+ENV CGO_CFLAGS_ALLOW="directory.*"
+ENV LD_LIBRARY_PATH=directory/lib:${LD_LIBRARY_PATH}
```

Comment:
Are you sure that `directory` shouldn't be, like, an actual directory?

Reply:
I planned to remove it, thanks for pointing it out! :+1: